projects/storefrontlib/cms-components/myaccount/my-coupons/claim-dialog/claim-dialog.component.html

```diff
@@ -0,0 +1,102 @@
+<div
+  class="cx-coupon-dialog"
+  role="dialog"
+  [cxFocus]="focusConfig"
+  (esc)="close('Escape clicked')"
+  aria-labelledby="dialogTitle"
+>
+  <div class="cx-coupon-container">
+    <!-- Modal Header -->
+
+    <div class="cx-dialog-header modal-header">
+      <h3
+        *cxFeature="'a11yDialogsHeading'"
+        id="dialogTitle"
+        class="cx-dialog-title modal-title"
+      >
+        {{ 'myCoupons.claimCoupondialogTitle' | cxTranslate }}
+      </h3>
+      <div
+        *cxFeature="'!a11yDialogsHeading'"
+        id="dialogTitle"
+        class="cx-dialog-title modal-title"
+      >
+        {{ 'myCoupons.claimCoupondialogTitle' | cxTranslate }}
+      </div>
+      <button
+        type="button"
+        class="close"
+        title="{{ 'common.close' | cxTranslate }}"
+        [attr.aria-label]="'common.close' | cxTranslate"
+        (click)="close('Cross click')"
+      >
+        <span aria-hidden="true">
+          <cx-icon [type]="iconTypes.CLOSE"></cx-icon>
+        </span>
+      </button>
+    </div>
+    <!-- Modal Body -->
+    <div class="cx-dialog-body modal-body">
+      <form (ngSubmit)="onSubmit()" [formGroup]="form" class="cx-dialog-item">
+        <label>
+          <span class="label-content"
+            >{{ 'myCoupons.claimCouponCode.label' | cxTranslate }}
+            <ng-template [ngTemplateOutlet]="requiredAsterisk"></ng-template
+          ></span>
+          <input
+            required="true"
+            type="text"
+            name="couponCode"
+            formControlName="couponCode"
+            placeholder="{{
+              'myCoupons.claimCouponCode.placeholder' | cxTranslate
+            }}"
+            class="form-control"
+          />
+
+          <!-- TODO: (CXSPA-7315) Remove feature toggle in the next major -->
```

Comment:
MAJOR: Comment with toggle could be removed, please review whole file, there are redundant feature toggles below also

Reply:
thanks, fixed.